server/controllers/article.js

```diff
@@ -259,3 +259,39 @@ export const deleteArticle = async (req, res) => {
     return serverError(res, error);
   }
 };
+
+export const rateArticle = async (req, res) => {
+  const { slug, user } = req;
+  const article = await Articles.findOne({ where: { slug } });
+  const articleId = article.get("id");
+  const raterId = user.userId;
+  const { score } = req.body;
+  const validScore = score < 6 && score > 0;
+  const isRating = await Ratings.findOne({
```

Comment:
please wrap this in a try and catch block

Reply:
Thanks, implemented